hotshot-state-prover/src/circuit.rs

```diff
@@ -476,9 +448,8 @@ mod tests {
             .is_err());
 
         // bad path: total weight doesn't meet the threshold
-        // bit vector with total weight 23
         let bad_bit_vec = [
-            true, true, true, true, true, false, false, true, false, false,
+            false, false, true, false, true, false, false, true, false, false,
```

Comment:
can you explain this change a bit?

Reply:
ah, the making sure the accumulated weight of the signers is lower than threshold. previously we set a higher threshold, now it's computed as 1/3 +1, so I let fewer nodes to vote.